mysql-test/suite/encryption/t/innodb_encryption_discard_import.test

```diff
@@ -124,3 +125,56 @@ FLUSH TABLES t1, t2, t3 FOR EXPORT;
 UNLOCK TABLES;
 
 DROP TABLE t1, t2, t3;
+
+--echo #
+--echo # MDEV-34670 IMPORT TABLESPACE unnecessary traverses
+--echo # 		tablespace list
+--echo #
+SET GLOBAL innodb_encrypt_tables= OFF;
+SET GLOBAL innodb_encryption_threads= 0;
+
+CREATE TABLE t1(f1 int,f2 text)ENGINE=InnoDB;
+INSERT INTO t1 VALUES(1, "InnoDB");
+CREATE TABLE t2 LIKE t1;
+ALTER TABLE t2 DISCARD TABLESPACE;
+FLUSH TABLES t1 FOR EXPORT;
+--copy_file $MYSQLD_DATADIR/test/t1.ibd $MYSQLD_DATADIR/test/t2.ibd
+UNLOCK TABLES;
+ALTER TABLE t2 IMPORT TABLESPACE;
```

Comment:
We should also rename the `t1.cfg` to `t2.cfg` in order to avoid a warning on `IMPORT`.